pkgcore/ebuild/atom.py

```diff
@@ -11,6 +11,7 @@
 __all__ = ("atom", "transitive_use_atom")
 
 import string
+from itertools import product
```

Comment:
Thanks =) Doesn't pmerge always redirect all the work to the transitive_use_atom.evaluate_conditionals ? which then just calculate conditional use dependency flags based on use flags provided by a parent. Or have I missed something I suspect might be related to 2 ?
I only see usage of that functionality in two places:
1. pquery while printing the revdeps of the package and intersecting with generated atoms(with all possible permutations of conditional use flags)
2. requesting for attribute change in package wrapper and then attempt force_{True,False} essentially trying to make corresponding restriction True or False.
Could you provide some examples where the second case might be applied/needed ? like forcing some attribute to change it's state despite having some initial configuration.